frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -150,6 +169,14 @@ export const rerunPipeline: KebabAction = (kind: K8sKind, pipelineRun: PipelineR
   };
 };
 
+export const rerunPipelineAndStay: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
+  return rerunPipeline(kind, pipelineRun);
+};
+
+export const rerunPipelineAndRedirect: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
+  return rerunPipeline(kind, pipelineRun, null, { onComplete: handlePipelineRunSubmit });
+};
+
```

Comment:
Why this change? I think flag thing (pipelineRunRedirect) worked fine.
You are still adding a check
`typeof onComplete === 'function' ? onComplete : () => {}`

Reply:
By default we can keep the flag true or false depending on majority usage of flag and make `customData` optional. WDYT?